### DISCUSSION TITLE
WPF controls or custom WPF library usage in MAUI

### DISCUSSION THREAD

**Discussion by vinaygupta31** [dotnet/maui#12298] [General]
Hi All,

We have some libraries written in WPF and current application UI is also in WPF & WinForms.
Going forward we want to use MAUI for a new software component but is it possible to reuse WPF libraries in that.
Currently we are targeting Windows IOT platform but going forward it can be cross platform as well. (Future Enahcements)

Is there any we can achieve it in MAUI ?

**Comment by xtuzy** [comment]
maui use winui3 on windows, winui3 not support wpf, so you can't use wpf control in maui
